changelogs/unreleased/8464-bump-pg-version-to-16.yml

```
@@ -0,0 +1,7 @@
+description: Bump postgresql version to 16
+issue-nr: 8464
+issue-repo: inmanta-core
+change-type: major
+destination-branches: [master]
+sections:
+  feature: "{{description}}"
```

```suggestion
  other-note: "{{description}}"
```

I'm honestly not sure, but I feel like "feature" is out of place.

---

Or upgrade-note?

---

> I'm honestly not sure, but I feel like "feature" is out of place.

Definitely :+1:

---

I think upgrade-note is probably best since this is not something we manage ourselves. And then perhaps reword the message to be a bit more user-oriented (we didn't "bump" the version)